src/Microsoft.AspNet.Identity.EntityFramework/IdentityUser.cs

```diff
@@ -87,5 +88,20 @@ public IdentityUser(string userName) : this()
         ///     Used to record failures for the purposes of lockout
         /// </summary>
         public virtual int AccessFailedCount { get; set; }
+
```

Comment:
The way they are will work fine, but they don't need to be virtual (although you may want them to be virtual for other reasons) and they don't need to have a setter.